OCR/ocr/reportvision-dataset-1/medical_report_import.py

```diff
@@ -0,0 +1,57 @@
+import os
+import json
+from datasets import load_dataset
+import random
+
+
+destination_folder = "reportvision-dataset-1"
+
+dataset = load_dataset("singhsays/fake-w2-us-tax-form-dataset")
+
+
+if not os.path.exists(destination_folder):
+    os.makedirs(destination_folder)
+
+images_folder = os.path.join(destination_folder, "images")
+json_folder = os.path.join(destination_folder, "ground_truth")
+
+os.makedirs(images_folder, exist_ok=True)
+os.makedirs(json_folder, exist_ok=True)
+
+generated_numbers = set()
+
+
+def generate_unique_random_number():
+    while True:
+        random_number = random.randint(100000, 999999)  # Generates a random unique 6-digit number
+        if random_number not in generated_numbers:
+            generated_numbers.add(random_number)
+            return random_number
+
+
+for split in dataset.keys():
+    split_data = dataset[split]
+    for example in split_data:
+        unique_id = generate_unique_random_number()
```

Comment:
That is a good point. Originally I was keep the unique id's because the files kept getting overwritten. I have edited the script to generate Id's sequentially this should not be an issue anymore.